modular_skyrat/modules/modular_clothing/armor/armor.dm

```diff
@@ -0,0 +1,72 @@
+//TECHARMOR
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor //No, this isn't spaceproof. Yes, the sprite reflects that this is not spaceproof properly. I made it a hardsuit subtype to make things easier and more readable.
+	name = "type I techhelmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	flash_protect = FLASH_PROTECTION_FLASH //The visor protects from flashes, but you're still going to go blind if you try to weld something.
+	icon_state = "hardsuit-secexo"
+	icon_state = "hardsuit0-secexo"
+	inhand_icon_state = "hardsuit0-secexo"
+	hardsuit_type = "secexo"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
+	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEMASK|HIDEEARS|HIDEFACE|HIDEHAIR
+	mutant_variants = STYLE_MUZZLE
+
+/obj/item/clothing/suit/space/hardsuit/security_armor
+	name = "type I full-body techarmor"
+	desc = "A specialized exoskeleton armor suit, comprised of flexible protective shielding. Comes equipped with a retractable helmet which offers a flash-resistant HUD visor, along with a headlamp."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "hardsuit-secexo"
+	inhand_icon_state = "hardsuit-secexo"
+	max_integrity = 250
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
```

Comment:
This should have ```'wound' = 10``` tacked onto the end. This way it shares the value with the closest approximate. The armor vest. Same with the other armors.

Additionally, the armor should probably share values with its closest approximate. The armor vest. Which would be, adjusted for it being an environment suit

```armor = list(MELEE = 35, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 100, RAD = 0, FIRE = 50, ACID = 75, WOUND = 10)```

This isn't as outrageous as it seems. The acid and fire damage is only significant for the armor itself taking damage, the bio doesn't actually do anything and there was a balance change across all armors that made energy armor (the damage disablers and ebows do) +10 of the clothings laser armor, with the only exception being the ablative armor since it deflects energy attacks.

This value should also be shared by the helmet.

Reply:
Values were from the old base and are supposed to be shared between both sets; fixing now.